### DISCUSSION TITLE
Error: Error loading ava.config.js: require is not defined

### DISCUSSION THREAD

**Discussion by Maximilian5189** [avajs/ava#2656] [Q&A] [answered]
After cloning the repository `git clone https://github.com/avajs/ava.git` and installing the dependencies with `npm i`, I wanted to run `npm test` as described in the contributing guidline (https://github.com/avajs/ava/blob/master/.github/CONTRIBUTING.md)

Then however, I encountered the following error:
`Error: Error loading ava.config.js: require is not defined
    at loadJsConfig (/Users/maximilian/node_modules/ava/lib/load-config.js:63:23)
    at loadConfig (/Users/maximilian/node_modules/ava/lib/load-config.js:204:3)
    at async resolveGlobs (/Users/maximilian/node_modules/ava/eslint-plugin-helper.js:78:29)
    at async MessagePort.handleMessage (/Users/maximilian/node_modules/ava/eslint-plugin-helper.js:136:19)`

Am I missing something? In the contribution guide I couldn't find if I have to perform any additional step. Thanks in advance.

**Comment by novemberborn** [comment, marked as answer]
What's your Node.js version?

Does it help if you clone the repo outside of a `node_modules` directory?

**Reply by Maximilian5189** [reply]
With `node -v` I get `v14.15.0`.

Yes I created the repo outside `node_modules` and wanted to run the AVA tests itself.

What I did: 
`cd ~ && mkdir avajs`
`cd avajs &&  git clone https://github.com/avajs/ava.git`
`npm i && npm test`

**Reply by novemberborn** [reply]
With your code sample, AVA ends up in `avajs/ava/`, but you run `npm test` from `avajs/`. Any chance you have a `package.json` higher up in your file system, with an `ava.config.js` next to it?

**Reply by Maximilian5189** [reply]
Indeed I overlooked one node_modules folder higher up in the file system. It works now, many thanks.
